SITCOMTN-031.tex

```diff
@@ -71,27 +71,29 @@ \section{Working Group Scope}
 The working group will deliver their report for May 31st 2022.
 
 \section{Charge}
-The working group will answer the following questions:
+The working group will answer the following questions in a form of a report:
 \begin{enumerate}
-\item{Summarize the status of the workflow: 24h definition, Jira OBS project, operation rehearsals}
-\item{What information is needed to be conveyed from one activity to another? This includes the definition of the observing plans, the down stream processing, required fixes...} 
-\item{What are the roles and responsibilities of the people in charge of transferring information and making decision at a given time?}
-\item{What tooling (existing or new) are needed  to enable this workflow?}
-\item{How do we handle the notebooks created for commissioning / testing? What is the process to archive them?}
+\item{Summarize the status of the workflow using existing documentation: 24h definition, Jira OBS project, operation rehearsals}
+\item{Defines what information is needed to be conveyed from one activity to another? This includes the definition of the observing plans, the down stream processing, required fixes...} 
+\item{Define the roles and responsibilities of the people in charge of transferring information and making decision at a given time?}
+\item{List the tooling (existing or new) needed  to enable this workflow?. This includes the way we will handle the notebooks created for commissioning / testing and the process to archive them?}
 \end{enumerate}
 
+It might be useful to create different reports depending on the cycle length defined in the scope of work.
+
 In the execution of this charge the Working Group is requested to reach out to other stakeholders in their deliberations and evaluations.
 The outcome from this working will be a document that reports on the charge items indicated above.
 
-\section{Working Group Members}
+\section{Proposed Working Group Members}
 \begin{enumerate}
-\item Patrick Ingraham (Chair)
-\item Robert
-\item Frossie
-\item Alysha
-\item Holger
+\item Patrick Ingraham (Chair): SIT-Com coordinator 
+\item Robert: SIT-Com coordinator
```

Comment:
I don't think both Robert and I need to be on this....

Reply:
no longer applies, please re-review.